vnext/Scripts/rnw-dependencies.ps1

```diff
@@ -68,6 +68,11 @@ $requirements = @(
         Name = 'Developer mode is on';
         Valid = try { (Get-WindowsDeveloperLicense).IsValid } catch { $false }
     },
+    @{
+        Name = 'Long path support is enabled';
+        Valid = try { (Get-ItemProperty HKLM:/SYSTEM/CurrentControlSet/Control/FileSystem -Name LongPathsEnabled).LongPathsEnabled -eq 1} catch { $false };
```

Comment:
(nit)

Why forward instead of backward slashes?

Reply:
doesn't really make a difference to PS AFAIK, do you see any issues using / instead of \ ?